### ISSUE TITLE
Electrum closes/crashes after entering password for wallet

### ISSUE THREAD

**Issue by z945** [spesmilo/electrum#2676]
Electrum-2.9.0
win7 pro sp1, installation and creating a new wallet, crashes after entering password

**Comment by ecdsa** [comment]
is this reproducible?

**Comment by z945** [comment]
yes, i  installed and deleted 4 or 5 times, tried old versions, dont work on my system

**Comment by ecdsa** [comment]
does the crash depend on the password?

**Comment by z945** [comment]
no, i tried complex and simple

**Comment by ecdsa** [comment]
ok, remove the "config" file of electrum, and try again

**Comment by z945** [comment]
On another laptop, win7 sp1 64 everything went right, i logon normal.
It can depend on the antivirus?

**Comment by ecdsa** [comment]
no, it is unrelated. please try to do what I said

**Comment by ecdsa** [comment]
also note that we made a new release this morning

**Comment by z945** [comment]
I'll try to put a new release on my home laptop. Thank you.

**Comment by ecdsa** [comment]
again: please try to remove the config file.

**Comment by z945** [comment]
>>again: please try to remove the config file.
The problem is not in the configuration file.

The problem is the wallet was on the disk c: \\
I moved the folder with wallet(Roaming\Electrum) from the c: \\ drive to another local disk, and  enter to the wallet.
On the home computer i have Avira antivirus with default settings and apparently it blocked access when the wallet was on the c: \\ drive. Or another system of security, but I did not specifically configure the lock for the system disk.
